README.md

```diff
@@ -16,8 +16,8 @@ This is a [Gradle](http://www.gradle.org/) plugin to allow easily performing Jav
     * Currently tested against Gradle 3.0-3.5.1 and 4.0-4.10.2
     * If you need support for Gradle 2.0-2.14.1, version 0.9.1 was the last version tested for compatibility; please see [the Gradle plugin portal](https://plugins.gradle.org/plugin/com.commercehub.gradle.plugin.avro)
     * Other versions may be compatible, but Gradle 1.x versions are unlikely to work
-* Currently built against Avro 1.8.2
-    * Currently tested against Avro 1.8.2; other versions may be compatible
+* Currently built against Avro 1.9.0-SNAPSHOT
+    * Currently tested against Avro 1.9.0-SNAPSHOT; other versions may be compatible
```

Comment:
This change should be reverted

Reply:
Hmm, I see that we should add a "If you need support for Avro 1.8.2, try plugin version x" line, but why would we reverse this change entirely?

Reply:
Oops.  I misread this as still being part of CHANGES.md.  You are totally correct.  We should add a section as you suggested, to indicate that people who want 1.8.2 support should use version 0.16.0.

Reply:
Done.